storage/mysql/tree_storage.go

```diff
@@ -244,6 +247,15 @@ func (t *treeTX) getSubtrees(ctx context.Context, treeRevision int64, nodeIDs []
 			subtree.Prefix = []byte{}
 		}
 		ret = append(ret, &subtree)
+
+		if glog.V(4) {
+			glog.Infof("  subtree: NID: %x, prefix: %x, depth: %d",
+				subtreeIDBytes, subtree.Prefix, subtree.Depth)
+			for k, v := range subtree.Leaves {
+				b, _ := base64.StdEncoding.DecodeString(k)
```

Comment:
If you've got debugging logging on at this point, you'd probably want to know if one of your key string is invalid base64 vs an empty string - that could be _days_ of hair pulling :)